app/models/form526_submission.rb

```diff
@@ -93,12 +93,7 @@ def submit_with_birls_id_that_hasnt_been_tried_yet!(
   )
     untried_birls_id = birls_ids_that_havent_been_tried_yet.first
 
-    # If there are no more ids to try, queue backup (if enabled), and return. End of the road, do not retry.
-    unless untried_birls_id
-      # hits this when it has a non-retryable error and has exhausted all birls
-      queue_central_mail_backup_submission_for_non_retryable_error!
-      return
-    end
+    return unless untried_birls_id
```

Comment:
This is a bit of a confusing pattern.  Is there a reason we don't move this condition to where this method is called?  Multiple returns in a single method is generally considered a ruby anti-pattern